Dockerfile

```diff
@@ -13,9 +13,9 @@ ENV SSUI_ROOT /opt/manageiq/manageiq-ui-self_service
 
 # Fetch postgresql 9.4 COPR and pglogical repos
 RUN curl -sSLko /etc/yum.repos.d/rhscl-rh-postgresql94-epel-7.repo \
-https://copr-fe.cloud.fedoraproject.org/coprs/rhscl/rh-postgresql94/repo/epel-7/rhscl-rh-postgresql94-epel-7.repo && \
-curl -sSLko /etc/yum.repos.d/ncarboni-pglogical-SCL-epel-7.repo \
-https://copr.fedorainfracloud.org/coprs/ncarboni/pglogical-SCL/repo/epel-7/ncarboni-pglogical-SCL-epel-7.repo
+      https://copr-fe.cloud.fedoraproject.org/coprs/rhscl/rh-postgresql94/repo/epel-7/rhscl-rh-postgresql94-epel-7.repo && \
+    curl -sSLko /etc/yum.repos.d/ncarboni-pglogical-SCL-epel-7.repo \
```

Comment:
The `https` is a continuation of the above line, the `curl` is a separate command